base64/Tiltfile

```diff
@@ -0,0 +1,9 @@
+def encode_base64(content):
+    command_value = 'echo {} | base64'.format(content)
```

Comment:
With longer strings, base64 will insert newlines every 76 characters. `base64 -w 0` avoids that.